src/sidebar/store/modules/selection.js

```diff
@@ -441,12 +441,18 @@ function focusModeFocused(state) {
 }
 
 /**
- * Returns the username for a focused user or `null` if no focused user.
+ * Returns the `userid` for a focused user or `null` if no focused user.
  *
  * @return {object|null}
  */
-function focusModeUsername(state) {
+function focusModeUserId(state) {
   if (
```

Comment:
I suggest this function would be easier to read if `state.selection.focusMode.config.user` were extracted and tested once at the start.